coreos-base/coreos/coreos-0.0.1.ebuild

```diff
@@ -141,9 +139,13 @@ RDEPEND="${RDEPEND}
 	net-misc/wget
 	net-misc/whois
 	net-vpn/wireguard-tools
+	sec-policy/selinux-virt
+	sec-policy/selinux-base
+	sec-policy/selinux-base-policy
```

Comment:
note: we can also add `sec-policy/selinux-unconfined`

Reply:
Done, I'm still having the problem that the selinux use flag doesn't get applied on arm64 (dbus, systemd, and I think even tar and other packages)

Reply:
@pothos : Have a look at `profiles/coreos/amd64/generic/use.mask` - `selinux` is masked there.